ui/app/styles/components/bar-chart.scss

```diff
@@ -0,0 +1,68 @@
+.bar-chart-wrapper {
+  border: $light-border;
+  border-radius: $radius-large;
+  padding: $spacing-l $spacing-l $spacing-s $spacing-l;
+  height: 100%;
+  width: 100%;
+
+  > div.is-border {
+    border: 0.3px solid $ui-gray-200;
+    width: 94%;
+    margin-left: 3%;
+    margin-bottom: $spacing-xxs;
+  }
+}
+
+.chart-header {
+  margin-left: $spacing-l;
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+
+  .header-left {
+    .chart-title {
+      font-size: $size-5;
+      font-weight: $font-weight-bold;
+      line-height: normal;
+    }
+
+    .chart-description {
+      font-size: $size-8;
+      font-weight: $font-weight-normal;
+      color: $ui-gray-700;
+      margin-bottom: $spacing-xs;
+    }
+  }
+
+  .header-right {
```

Comment:
`.header-right` is where anything passed into the block is yielded (i.e. an export button). Should this styling be less specific since the button is no longer part of this component?